services/freelance-matching/.env.example

```
@@ -0,0 +1,19 @@
+# Database
+DATABASE_URL="prisma+postgresql://username:password@localhost:5432/freelance_matching?schema=public"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)